lib/stdlib/src/rand.erl

```diff
@@ -44,11 +47,12 @@
 %% This depends on the algorithm handler function
 -type alg_seed() :: exs64_state() | exsplus_state() | exs1024_state().
 %% This is the algorithm handler function within this module
--type alg_handler() :: #{type      := alg(),
-			 max       := integer(),
-			 next      := fun(),
-			 uniform   := fun(),
-			 uniform_n := fun()}.
+-type alg_handler() :: #{type      => alg(),
```

Comment:
Why are you making the keys optional in `alg_handler()`? It seems to me that all of these keys are always present, and some of them must be to make the interface functions not crash. `export_seed_s/1` crashes if `type` isn't present, `uniform_s/1` crashes if `uniform` isn't present, `jump/1` crashes if `jump` isn't present, and `normal_s/1` crashes (in `get_52/1`) if `next` isn't present.